cmd/job_list.go

```diff
@@ -98,12 +146,32 @@ func renderItemDetails(item *svc.ListJobItem) (details []string) {
 	if item.Details.UnschedulableReason != "" {
 		usreason := item.Details.UnschedulableReason
 		if strings.Contains(usreason, "NotYetSchedulable") {
-			usreason = "No resources available"
+			//this is only shown when there is no quota configured
+			//@TODO what if the quota is large enough but a job will never fit on a node?
+			usreason = "Not enough cluster resources"
 		}
 
 		details = append(details, usreason)
 	}
 
+	if len(item.Details.FailedCreateEvents) > 0 {
+		evreasons := ""
+		for _, ev := range item.Details.FailedCreateEvents {
+			if strings.Contains(ev.Message, "exceeded quota") && quota != nil {
+				if item.Memory > quota.RequestMemory || item.VCPU > quota.RequestCPU {
+					//user has specified something that will never fit with the curren quota settings
+					evreasons = "Specified job size exceeds current resource maximum"
```

Comment:
I find it a bit confusing to talk about `job size`, maybe it would be better to say `job resource request`?